clusters/build-clusters/02_cluster/prow/03_deployment/admin_crier_rbac.yaml

```diff
@@ -0,0 +1,43 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: crier
+    namespace: ci
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: crier
+    namespace: ci
+  rules:
+    - apiGroups:
+        - prow.k8s.io
+      resources:
+        - prowjobs
```

Comment:
The build02 cluster doesn't have prowjob resources and crier only needs to access pods and events on it

Reply:
Removed.